app.js

```diff
@@ -124,9 +125,65 @@ app.post("/rescheduling_request", function (req, res) {
   console.log("Request Received: POST a reschedulling request");
 });
 
-// Empty route to POST feedback requests.
+/**
+ * Creates a new Feedback record.
+ *
+ * The POST request to this endpoint should hold 2 parameters:
+ * class_ID: the ID of the class that is requested to be cancelled
```

Comment:
This comment needs to be edited

Reply:
@absirgue You still didn't change this. Check the class_ID comment, it shouldn't be "cancelled"